android/ironoxide-android/build.gradle

```diff
@@ -5,6 +5,7 @@ apply plugin: 'signing'
 android {
     compileSdkVersion 29
     buildToolsVersion "29.0.3"
+    ndkVersion "21.3.6528147"
```

Comment:
This is the NDK version that the GHA runner had installed when it tried to build our AARs. If a user does not have this version installed, the error message will lead them to installing the correct version.